pennylane/kernels/postprocessing.py

```diff
@@ -0,0 +1,189 @@
+    """
+    w, v = np.linalg.eigh(K)
+
+    if np.min(w) < 0:
+        # Transform spectrum: absolute value
+        w0 = np.abs(w)
+
+        return (v * w0) @ np.transpose(v)
+
+    return K
+
+
+def closest_psd_matrix(K, fix_diagonal=False, solver=None, **kwargs):
+    r"""Return the closest positive semidefinite matrix to the given kernel matrix.
+
+    This method optionally achieves the correct diagonal entries
+    (``fix_diagonal=True``) or keeps the eigenvectors intact (``fix_diagonal=False``).
+    For ``fix_diagonal=False``, this method reduces to the ``threshold_matrix`` method.
+
+    Args:
+        K (array[float]): Kernel matrix assumed to be symmetric.
+        fix_diagonal (bool): Whether to fix the diagonal to 1. Defaults to False.
+        solver (str, optional): Solver to be used by cvxpy. Defaults to CVXOPT.
+        kwargs (kwarg dict): Passed to cvxpy.Problem.solve()
+
+    Returns:
+        array[float]: closest positive semidefinite matrix in Frobenius norm.
+
+    Comments:
+        Requires cvxpy and the used solver (default CVXOPT) to be installed if ``fix_diagonal=True``.
+
+    Reference:
+        This method is introduced in `arXiv:2105.02276 <https://arxiv.org/abs/2105.02276>`_.
+    """
+    if not fix_diagonal:
+        return threshold_matrix(K)
+    try:
+        import cvxpy as cp
+
+        if solver is None:
+            solver = cp.CVXOPT
+    except ImportError as e:
+        raise ImportError("CVXPY is required for this post-processing method.") from e
+
+    X = cp.Variable(K.shape, PSD=True)
+    constraint = [cp.diag(X) == 1.0] if fix_diagonal else []
+    objective_fn = cp.norm(X - K, "fro")
+    problem = cp.Problem(cp.Minimize(objective_fn), constraint)
+
+    try:
+        problem.solve(solver=solver, **kwargs)
+    except Exception:
+        try:
+            problem.solve(solver=solver, verbose=True, **kwargs)
+        except Exception as e:
+            raise RuntimeError("CVXPY solver did not converge.") from e
+
+    return X.value
+
+
+def mitigate_depolarizing_noise(K, num_wires, method, use_entries=None):
+    r"""Estimate depolarizing noise rate(s) using on the diagonal entries of a kernel
+    matrix and mitigate the noise, assuming a global depolarizing noise model.
+
+    Args:
+        K (array[float]): Noisy kernel matrix.
+        num_wires (int): Number of wires/qubits of the quantum embedding kernel.
+        method ('single'|'average'|'split_channel'): Strategy for mitigation
+
+            * 'single': An alias for 'average' with len(use_entries)=1.
+            * 'average': Estimate a globale noise rate based on the average of the diagonal
+              entries in use_entries.
+            * 'split_channel': Estimate individual noise rates per embedding.
+        use_entries=None (array[int]): Diagonal entries to use if method in ['single', 'average'].
+            If None, defaults to [0] ('single') or range(len(K)) ('average').
```

Comment:
```suggestion
        use_entries (array[int]): Diagonal entries to use if method in ``['single', 'average']``.
            If ``None``, defaults to ``[0]`` (``'single'``) or ``range(len(K))`` (``'average'``).
```

No need to include defaults here, as Sphinx will automatically extract them from the signature (leading to duplication)